README.md

```diff
@@ -1,2 +1,69 @@
-# weatherApp
-As part of Microverse curriculum
+# Weather APP
+Weather App using API
+
+> This is an assigment for Microverse JavaScript course. The goal is to create a website using JavaScript API to show weather in the city.
+
+![screenshot](src/assets/Screenshot_1.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML,
+- CSS,
+- JavaScript,
+- npm,
+- webpack, 
+- OpenWeatherApi
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- A compatible browser with HTML, CSS and JavaScript. 
+
+### Install
+
+```
+- git clone https://github.com/paulzay/weatherapp;
+- cd weatherapp/
+- npm install
+- npm webpack --watch
+```
+
```

Comment:
## Add a live link to the page

- [ ] Add a live link to where the page is live. let's make sure the README is more descriptive. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))